mllib/src/test/scala/org/apache/spark/ml/feature/Word2VecSuite.scala

```diff
@@ -188,6 +188,15 @@ class Word2VecSuite extends SparkFunSuite with MLlibTestSparkContext with Defaul
     assert(math.abs(similarity(5) - similarityLarger(5) / similarity(5)) > 1E-5)
   }
 
+  test("Word2Vec read/write numPartitions calculation") {
```

Comment:
Should this test hardcode a specific ``spark.kryoserializer.buffer.max``? That could allow us to be more explicit in the assertions